### PR TITLE
docs: contributing get started new content

### LINKED ISSUE #3621: Website docs: Getting started page
- [x] Update the Getting started page guidance in a PR
This issue is a complete re-write of the /contribution/overview page.

All headers contain new content:

Getting started
Contribution gallery (updates to the Airtable iframe can be tracked https://github.com/carbon-design-system/carbon-website/issues/3718 )
Carbon champions (reviewers, please verify names and work are accurate. We are getting permission to display names)
Types of contribution
The process

- [x] Mid sprint review

---

- Ref Figma prototypes [here](https://www.figma.com/file/9pdTqNPVJCmDOAacm3Rnxp/Contribution?type=design&node-id=1-3128&mode=design&t=iGRdISw6LiZGQqwa-0)

### PR DESCRIPTION
**Pull request by aubrey-oneal**
Closes #3621

This issue is a complete re-write of the /contribution/overview page. 

#### Changelog

**New**

All headers contain new content:
- Getting started
- Contribution gallery (updates to the Airtable iframe can be tracked #3718 )
- Carbon champions (We removed the names until we get permission to display them #3753 to unblock this PR)
- Types of contribution
- The process

### CODE REVIEW

File: src/pages/contributing/get-started.mdx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+make a change, commit and push to your branch normally. Once all revisions to
+your pull request are complete, a maintainer will squash and merge your commits
+for you.
+
+For work that does not have a code deliverable, such as revisions to Figma
+libraries, ask the Carbon team how to finish your contribution.
+
+### Contributor License Agreement
+
+The Carbon core team works for IBM. To accept contributions, we need a signed
```
Comment (alisonjoseph) [inline src/pages/contributing/get-started.mdx:161]:
```suggestion
The Carbon core team is part of IBM. To accept contributions, we need a signed
```

Not sure my suggestion is better, but "works" for makes it feel like we are outside contractors.

File: src/pages/contributing/get-started.mdx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+Most contributors work in groups of 2-3 and either set up weekly sessions or
+join [meetups](https://carbondesignsystem.com/whats-happening/meetups/) such as
+the Figma Guild, Data Viz Guild, or office hours. In these sessions, it is
+common to share work in progress and ask lots of questions. As you make
+progress, update your GitHub issue.
+
+### Review and published
+
+Most work that results in code will be reviewed directly in a pull request.
```
Comment (alisonjoseph) [inline src/pages/contributing/get-started.mdx:149]:
```suggestion
Work that results in code will be reviewed directly in a pull request.
```
Not sure we need "most", could also be "All work." or "Any work.."

File: src/pages/contributing/get-started.mdx [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+[existing GitHub issues](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22)
+labeled with `community contribution`. You can also filter by role:
+[developer](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22++label%3A%22role%3A+dev+%F0%9F%A4%96%22_)
+or
+[designer](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22++label%3A%22role%3A+design+%3Apencil2%3A%22+).
+Put a comment in the issue saying you'd like the help.
+
+The Carbon team is also happy to help you find a good first issue. Just stop by
+our [office hours](https://carbondesignsystem.com/whats-happening/meetups/) or
+send us a slack message.
```
Comment (alisonjoseph) [inline src/pages/contributing/get-started.mdx:127]: Should this link somewhere, so they know how to send us a slack message? Maybe here? https://carbondesignsystem.com/help/contact-us/#slack-channels

**Review by alisonjoseph — COMMENTED** [review]
This is great, had a couple small comments

File: src/pages/contributing/get-started.mdx (hunk 2 above)
Comment (aubrey-oneal) [reply src/pages/contributing/get-started.mdx]: Thanks! I changed it to 'Work'

**Review by alisonjoseph — APPROVED** [review]

File: src/pages/contributing/get-started.mdx [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+<AnchorLink>Carbon champions</AnchorLink>
+<AnchorLink>Types of contribution</AnchorLink>
+<AnchorLink>The process</AnchorLink>
+
+</AnchorLinks>
+
+## Get started
+
+If you're interested in getting involved but don't know where to start, welcome!
+Designers and developers like you help to make Carbon great, and so we're glad
```
Comment (laurenmrice) [inline src/pages/contributing/get-started.mdx:29]: > Designers and developers like you help to make Carbon great, and so we’re glad you’re here.

Do you mean the above^ or do you mean "...and we're so glad you're here." Could be either but just wanted to double check with you.

File: src/pages/contributing/get-started.mdx [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+Check out the
+[status of design kits](https://www.figma.com/file/CFMtqV5Nztdbm0mi2UiDLg/Library-%2B-Template-Planning?type=design&node-id=3713-26762&mode=design&t=HqvAYXUeccKNMstT-0)
+and get involved by reaching out to the contacts listed.
+
+### Website documentation
+
+All components and patterns require usage, style, code, and accessibility
+guidance on the Carbon website. You can learn how to contribute documentation
+smoothly and utilize our markdown templates on the
+[documentation](contributing/documentation/) page. A list of components that
```
Comment (laurenmrice) [inline src/pages/contributing/get-started.mdx:96]: The "documentation" link gives me a 404.

File: src/pages/contributing/get-started.mdx [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+### Fixing issues and bugs
+
+If you find a bug in the source code, you can help us by
+[filling out a bug report](https://github.com/carbon-design-system/carbon/issues/new/choose)
+and, ideally, submitting a pull request with a fix.
+
+### Icons and pictograms
+
+You can find instructions on how to create and contribute
+[UI icons](https://www.ibm.com/design/language/iconography/ui-icons/contribute),
```
Comment (laurenmrice) [inline src/pages/contributing/get-started.mdx:109]: Should `App icons` and `Pictograms` also be capitalized in this case like they are on IDL or should they be lower case?

File: src/pages/contributing/get-started.mdx [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+
+#### Volunteer for existing work
+
+The best way to volunteer is to look through
+[existing GitHub issues](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22)
+labeled with `community contribution`. You can also filter by
+[developer](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22++label%3A%22role%3A+dev+%F0%9F%A4%96%22_)
+or
+[designer](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22++label%3A%22role%3A+design+%3Apencil2%3A%22+)
+roles. Put a comment in the issue saying you'd like the help.
```
Comment (laurenmrice) [inline src/pages/contributing/get-started.mdx:130]:
```suggestion
roles. Put a comment in the issue saying you'd like to help.
```

File: src/pages/contributing/get-started.mdx [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+
+The Carbon team is also happy to help you find a good first issue. Just stop by
+our [office hours](/whats-happening/meetups/) or send us a slack message.
```
Comment (laurenmrice) [inline src/pages/contributing/get-started.mdx:133]: You could link to the anchor link for Office hours here if you want it to be more direct.

File: src/pages/contributing/get-started.mdx [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+your pull request are complete, a maintainer will squash and merge your commits
+for you.
+
+#### Contributor License Agreement
+
+The Carbon core team is part of IBM. To accept contributions, we need a signed
+Contributor License Agreement (CLA) from you before code contributions can be
+reviewed and merged. You will be prompted to do this in your PR, but if you have
+questions, please don't hesitate
+to [reach out](help/contact-us/#slack-channels).
```
Comment (laurenmrice) [inline src/pages/contributing/get-started.mdx:170]: I am getting a 404 for this "reach out" link.

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Looks really good Aubrey! ⭐️ Just a couple things: 

Gallery airtable:
Thy/I can figure out if we should populate this list with some more items and fix some of the thumbnails to be more interactive.

**Comment by Huixh111** [comment]
o

File: src/pages/contributing/get-started.mdx (hunk 6 above)
Comment (aubrey-oneal) [reply src/pages/contributing/get-started.mdx:109]: I think they are only capitalized in the LHN but when I checked out the docs pages (i.e. design tab) they were lowercased so I will keep them as is.

File: src/pages/contributing/get-started.mdx (hunk 4 above)
Comment (aubrey-oneal) [reply src/pages/contributing/get-started.mdx:29]: Good catch! I meant the later.

File: src/pages/contributing/get-started.mdx (hunk 9 above)
Comment (aubrey-oneal) [reply src/pages/contributing/get-started.mdx]: thanks! I missed a '/'

File: src/pages/contributing/get-started.mdx (hunk 5 above)
Comment (aubrey-oneal) [reply src/pages/contributing/get-started.mdx]: Thanks! I missed a '/'

File: src/pages/contributing/get-started.mdx (hunk 6 above)
Comment (laurenmrice) [reply src/pages/contributing/get-started.mdx:109]: Got it!

**Review by laurenmrice — APPROVED** [review]

**Pull request merged**